src/main/java/uk/co/fivium/els/categories/airconditioners/model/HeatingDuctlessAirConditionersForm.java

```diff
@@ -0,0 +1,205 @@
+
+  @FieldPrompt("Is data available for warmer climate conditons?")
+  @NotNull(message = "Select whether data is available for warmer climate conditions")
+  private Boolean warmerConditions;
+
+  @FieldPrompt("Energy efficiency class for Warmer heating season")
+  @NotBlank(groups = WarmerClimateGroup.class, message = "Select an energy efficiency indicator for warmer climate conditions")
+  private String warmerHeatingEfficiencyRating;
+
+  @FieldPrompt("Design load for heating in warmer climate conditions in kW")
+  @Digits(groups = WarmerClimateGroup.class, integer = 2, fraction = 1, message = "Enter the design load for warmer climate conditions, up to 2 digits with an optional decimal place")
+  private String warmerHeatingDesignLoad;
+
+  @FieldPrompt("Seasonal coefficient of performance in warmer climate conditions (SCOP value)")
+  @Digits(groups = WarmerClimateGroup.class, integer = 1, fraction = 1, message = "Enter the SCOP value for warmer climate conditions, 1 digit with an optional decimal place")
+  private String warmerScop;
+
+  @FieldPrompt("Annual energy consumption in kWh per year in warmer climate conditions")
+  @Digits(groups = WarmerClimateGroup.class, integer = 4, fraction = 0,message = "Enter the annual energy consumption for warmer climate conditions, up to 4 digits")
+  private String warmerAnnualEnergyConsumption;
+
+  @FieldPrompt("Is data available for colder climate conditons?")
+  @NotNull(message = "Select whether data is available for colder climate conditions")
+  private Boolean colderConditions;
+
+  @FieldPrompt("Energy efficiency class for Colder heating season")
+  @NotBlank(groups = WarmerClimateGroup.class, message = "Select an energy efficiency indicator for colder climate conditions")
+  private String colderHeatingEfficiencyRating;
+
+  @FieldPrompt("Design load for heating in colder climate conditions in kW")
+  @Digits(groups = WarmerClimateGroup.class, integer = 2, fraction = 1, message = "Enter the design load for colder climate conditions, up to 2 digits with an optional decimal place")
+  private String colderHeatingDesignLoad;
+
+  @FieldPrompt("Seasonal coefficient of performance in colder climate conditions (SCOP value)")
+  @Digits(groups = WarmerClimateGroup.class, integer = 1, fraction = 1, message = "Enter the SCOP value for colder climate conditions, 1 digit with an optional decimal place")
+  private String colderScop;
+
+  @FieldPrompt("Annual energy consumption in kWh per year in colder climate conditions")
+  @Digits(groups = WarmerClimateGroup.class, integer = 4, fraction = 0,message = "Enter the annual energy consumption for colder climate conditions, up to 4 digits")
+  private String colderAnnualEnergyConsumption;
+
+  @FieldPrompt("Sound power levels for indoor units expressed in dB(A) re1 pW")
+  @Digits(integer = 2, fraction = 0, message = "Enter the outdoors sound power level, up to 2 digits long")
+  private String soundPowerLevelIndoors;
```

Comment:
Think we should probably move these questions to above the wamer/colder questions just to make it clear they are not related to a climate group